sdk/core/core-websockets/src/client.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
Is it worth also adding tracing for this just for better observability if we dont' have that already?

Reply:
Good point, I'll attempt to add it in a follow-up PR.